job_executor/adapter/local_storage.py

```diff
@@ -362,7 +362,7 @@ def archive_input_files(dataset_name: str):
     Archives the input folder files if not already archived
     """
     archive_dir = INPUT_DIR / f"archive/{dataset_name}"
-    move_dir = INPUT_DIR / f"{dataset_name}"
+    move_dir = INPUT_DIR / f"decrypted/{dataset_name}"
```

Comment:
If I am understanding this will archive the unpackaged files which is not ideal 🤔 The .tar should be archived rather than the unpackaged files

Reply:
Yes, you are correct. It would be more better if we move the archiving step before unpackaging. Then, we should always call the unpackage function if we retrieve files from the archive.